entry/src/main/js/default/pages/index/index.js

```diff
@@ -177,15 +178,37 @@ export default {
                 this.temperature = data.temperature;
                 this.humidity = data.humidity;
                 this.location = data.location;
-                this.weather = data.weather;
-                this.day = data.day;
                 this.alert_title = data.alert_title;
                 this.alert_desc = data.alert_desc;
             }
         })
+        fetch.fetch({
+            url:'https://weatherdbi.herokuapp.com/data/weather/bangalore',
+            responseType:"json",
+            method: 'GET',
+            success:function(resp)
+            {
+                data = JSON.parse(resp.data);
+            },
+            fail:(err,code) => {
+                console.log("fail data:"+ JSON.stringify(err));
+                console.log("fail code:"+ code)
+            },
+            complete: ()=>{
+                this.temperatureInCelcius = data.currentConditions.temp.c;
+                this.weather = data.currentConditions.comment;
+            }
+        })
+        const date = new Date();
+        const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
```

Comment:
for strings please add then to respective locale json files and ensure chinese translations are added